src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2556,6 +2556,32 @@ theorem not_extractLsb'_append_not_extractLsb'_eq_not_extractLsb' {x : BitVec w}
     congr 1
     omega
 
+/-- A sign extension of `x : BitVec w`
+equals high bits of either `0` or `1` depending on `x.msb`,
+followed by the low bits of `x`. -/
```

Comment:
Can this be linebroken at the end of 80 or 100 columns, rather than at seemingly arbitrary places.

Reply:
I don't think so, I line-broke it at readable sections, so there's one at the `,`, and one where the sentence naturally ends. Happy to take a re-phrasing that line-breaks at english stops :)

Reply:
Can we use a line-break-algorithm that aligns with what latex or any standard word-processor would do. Meaning, fill lines until we reach the end of the column and then break.